src/main/java/net/coderbot/iris/shaderpack/IdMap.java

```diff
@@ -34,16 +35,16 @@
 	private final Object2IntMap<Identifier> entityIdMap;
 
 	/**
-	 * a map that contains the identifier of an item to the integer value parsed in block.properties
+	 * A map that contains the identifier of an item to the integer value parsed in block.properties
 	 */
-	private Map<Identifier, Integer> blockPropertiesMap = Maps.newHashMap();
+	private Map<Identifier, Integer> blockPropertiesMap = new HashMap<>();
 
 	/**
-	 * a map that contains render layers for blocks in block.properties
+	 * A map that contains render layers for blocks in block.properties
 	 */
-	private Map<Identifier, RenderLayer> blockRenderLayerMap = Maps.newHashMap();
+	private Map<Identifier, RenderLayer> blockRenderLayerMap = new HashMap<>();
```

Comment:
Why these changes?

Reply:
Maps.newHashMap() is deprecated and should be replaced with new HashMap<>();